background.ts

```diff
@@ -935,6 +935,11 @@ ipcMainProxy.handle('show-snapshots-blocking-dialog', async(
 
   dialog.webContents.on('ipc-message', (_event, channel, args) => {
     if (channel === 'dialog/mounted') {
+      if (os.platform() !== 'darwin') {
+        mainWindow?.webContents.send('window/blur', true);
+        mainWindow?.setMenuBarVisibility(false);
```

Comment:
I'm not sure if I agree with the idea of toggling the menu bar visibility. What benefit does this bring us in regards to blurring the main window, shouldn't interacting with the menu bar be disabled through the modal dialog?

Reply:
Yeh, there is no benefits indeed. I'll revert it